src/pages/studyView/StudyViewPage.tsx

```diff
@@ -152,19 +174,29 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
                                     disableAddGenomicButton={this.props.routing.location.query.tab === StudyViewPageTabKeys.CLINICAL_DATA}
                                 />}
 
-                                <div className={styles.selectedInfo}>
+
+                                <Observer>
```

Comment:
i should probably nest this in another component.  Observer is here so that page component will no re-render on all the chart data promises